### PR TITLE
Feature/otc

### PR DESCRIPTION
**Pull request by ndonnan**
OTC settlement work

### CODE REVIEW

File: scripts/transferSettlementTemp/interchangeFeeCalculation.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef */
```
Comment (ndonnan) [inline scripts/transferSettlementTemp/interchangeFeeCalculation.js:1]: This should be removed

File: package.json [hunk 2]
```diff
@@ -24,46 +24,57 @@
...
         "@mojaloop/central-services-health": "10.6.0",
         "@mojaloop/central-services-logger": "10.6.0",
-        "@mojaloop/central-services-shared": "11.2.0",
+        "@mojaloop/central-services-shared": "11.3.6",
         "@mojaloop/central-services-stream": "10.6.0",
         "@mojaloop/ml-number": "8.2.0",
         "@now-ims/hapi-now-auth": "2.0.2",
         "async": "3.2.0",
         "async-retry": "1.3.1",
+        "bignumber.js": "9.0.0",
```
Comment (ndonnan) [inline package.json:44]: Not sure why this is being included?

File: src/models/settlement/facade.js [hunk 3]
```diff
@@ -758,7 +761,7 @@ const Facade = {
         const settlementData = await knex('settlement AS s')
           .join('settlementStateChange AS ssc', 'ssc.settlementStateChangeId', 's.currentStateChangeId')
           .join('settlementModel AS sm', 'sm.settlementModelId', 's.settlementModelId')
-          .select('s.settlementId', 'ssc.settlementStateId', 'ssc.reason', 'ssc.createdDate', 'sm.autoPositionReset')
+          .select('s.settlementId', 'ssc.settlementStateId', 'ssc.reason', 'ssc.createdDate', 'sm.autoPositionReset', 'sm.requireLiquidityCheck')
```
Comment (ndonnan) [inline src/models/settlement/facade.js:764]: Need to do. a select for updated to lock the participant positions before updating

**Review by ndonnan — COMMENTED** [review]
Issues to resolve before merging

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -0,0 +1,109 @@
+stages:
```
Comment (ndonnan) [inline .gitlab-ci.yml:1]: Remove from PR

**Comment by vgenev** [comment]
merged with CGS functionality

**Pull request closed without merging**